release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -2775,6 +2775,16 @@ After you run the command, the CNO collects `must-gather` logs that you can insp
 +
 (link:https://issues.redhat.com/browse/OCPBUGS-52367[OCPBUGS-52367])
 
+* There is a known issue with Gateway API that prevents the provisioning of internal load balancers on {aws-first}, {gcp-first}, and {azure-first} private clusters. Load balancers configured for internal use are incorrectly determined to be external:
++
+--
+** In an {aws-short} private cluster, the load balancer becomes stuck in the `pending` state and reports the error: `Error syncing load balancer: failed to ensure load balancer: could not find any suitable subnets for creating the ELB`.
+
+** In {gcp-short} and {azure-short} private clusters, the load balancer is provisioned, but the existence of an external IP address is unexpected.
```

Comment:
```suggestion
** In {gcp-short} and {azure-short} private clusters, the load balancer is provisioned, but an external IP address is  provisioned, which is unexpected for an internal load balancer.
```

Reply:
this is a great suggestion that adds clarity, thank you, i'll update the text